Vagrantfile

```diff
@@ -33,18 +35,22 @@
 # This Vagrantfile allows customisation of some aspects of the virtaual machine
 # See the customization options below for details.
 #
-# The options can be set either by prefixing the vagrant command, or by
-# exporting to the environment.
+# The options can be set either by prefixing the vagrant command, using
+# `.vagrant.yml`, or by exporting to the environment.
 #
 #   # Prefixing the command
 #   $ ALAVETELI_VAGRANT_MEMORY=2048 vagrant up
 #
+#   # .vagrant.yml
+#   $ echo "memory: 2048" >> .vagrant.yml
```

Comment:
I think its fairly standard, even in Alaveteli:

```
$ find . -name *.yml -d 1
./.codeclimate.yml
./.hound.yml
./.ruby-style.yml
./.travis.yml
```

I did wonder about an example file, but I had thought that this example was enough.

I can pop an example file in if you think it would help.